jspsych-uil-error.js

```diff
@@ -71,6 +71,32 @@ else {
             dialog.append(p);
         };
 
+        /**
+         * Display an error dialog.
+         *
+         * This function displays errors with the script. Which may
+         * help the developpers of the script.
+         *
+         * @param {string} a message that is put in a paragraph of the
+         *                 error dialog.
+         */
+        context.scriptError = function (message) {
+
+            let dialog = _dialog;
+
+            if (_dialog === null) {
+                _dialog = createErrorDialog();
+                dialog = _dialog;
+            }
```

Comment:
I see this bit of code somewhere else also; I'd say we could refactor this into a separate ``getOrCreateDialog`` method?

Could be it's own issue tho